update/cloudvm.go

```diff
@@ -0,0 +1,96 @@
+package update
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	infrastructure "github.com/ninech/apis/infrastructure/v1alpha1"
+	"github.com/ninech/nctl/api"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	res "k8s.io/apimachinery/pkg/api/resource"
+)
+
+type cloudVMCmd struct {
+	Name        string            `arg:"" default:"" help:"Name of the CloudVM instance to update."`
```

Comment:
This should not have an empty default, the `name` is always required for updates.

Reply:
done